### ISSUE TITLE
Allow disabling request logging

### ISSUE THREAD

**Issue by francisdb** [fastify/fastify-reply-from#232]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the feature has not already been requested

### 🚀 Feature Proposal

For each request the following is logged:

```
| 2022-02-17 22:16:41 | response received
| 2022-02-17 22:16:41 | fetching from remote server
```

We would like to disable these. Maybe it could even be linked to the fastify `disableRequestLogging: true` option.

### Motivation

For every request we get 2 extra log lines. That adds up quickly, also if we keep them we would like them to be more descriptive.

### Example

```js
import replyFrom from 'fastify-reply-from'
...
app.register(replyFrom, {
      base: OUR_ENDPOINT,
      disableRequestLogging: true,
})
```

**Comment by francisdb** [comment]
We might also want to customize these logs, but there might already be some hooks we can use for that?

**Comment by Fdawgs** [comment]
Closed by #233
